api/provider/aws/task_list.go

```diff
@@ -0,0 +1,151 @@
+package aws
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/config"
+	"github.com/quintilesims/layer0/common/errors"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+const MAX_TASK_IDS = 100
+
+func (t *TaskProvider) List() ([]models.TaskSummary, error) {
+	clusterNames, err := t.listClusterNames()
+	if err != nil {
+		return nil, err
+	}
+
+	taskCopies := map[string][]*ecs.Task{}
+	for _, clusterName := range clusterNames {
+		environmentID := delLayer0Prefix(t.Config.Instance(), clusterName)
+
+		running := "RUNNING"
+		tasks, err := t.listTasksByStatus(&running, &environmentID)
```

Comment:
[nit] perhaps rename `tasks` to `runningTasks`, so it's consistent with `stoppedTasks`